config/config.go

```diff
@@ -1070,13 +1070,16 @@ type StorageConfig struct {
 	// required for `/block_results` RPC queries, and to reindex events in the
 	// command-line tool.
 	DiscardABCIResponses bool `mapstructure:"discard_abci_responses"`
+
+	GenesisHash []byte `mapstructure:"genesis_hash"`
```

Comment:
Since the hash is a fixed number of bytes should it be constrained here ? `[size]byte`

Reply:
Forcing this causes test failures, I think the check in `run_node` that the length is 64 is sufficient

Reply:
makes sense